ci/after_additional_test.sh

```diff
@@ -28,12 +28,26 @@ convox apps | grep httpd2
 convox apps info httpd2 | grep running
 endpoint=$(convox env get ENDPOINT -a httpd2)
 fetch https://$endpoint | grep "It works"
+
 # postgres resource test
 ps=$(convox api get /apps/httpd2/processes | jq -r '.[]|select(.status=="running" and .name == "web")|.id' | head -n 1)
 convox exec $ps "/usr/scripts/db_check.sh" -a httpd2
+
+# mariadb resource test
+ps=$(convox api get /apps/httpd2/processes | jq -r '.[]|select(.status=="running" and .name == "web")|.id' | head -n 1)
```

Comment:
do we need to test these resources in the after/before shell files?
otherwise, i'd say let's just add it to `test.sh`

Reply:
it's for the resource persistence testing. i think other dbs also should be included for this type of test.
that's why i added it.
What do you think?

Reply:
I think it's to check if the resource is available, right?

Reply:
yes